itests/qtest-druid/src/main/java/org/apache/hive/kafka/SingleNodeKafkaCluster.java

```diff
@@ -175,24 +165,20 @@ public void createTopicWithData(String topic, List<byte []> events) {
     }
   }
 
-  public void createTopic(String topic) {
-    int sessionTimeoutMs = 1000;
-    ZkClient zkClient = new ZkClient(
-        this.zkString, sessionTimeoutMs, sessionTimeoutMs,
-        ZKStringSerializer$.MODULE$
-    );
-    ZkUtils zkUtils = new ZkUtils(zkClient, new ZkConnection(zkString, sessionTimeoutMs), false);
+  private void createTopic(String topic) {
+    Properties properties = new Properties();
+    properties.setProperty("bootstrap.servers", "localhost:9092");
+    properties.setProperty("key.deserializer", "org.apache.kafka.common.serialization" 
```

Comment:
Maybe optimizer fixes this automatically, but could we just break the line at the ',' instead of adding a new concatenation?